package.json

```diff
@@ -28,6 +28,7 @@
   "license": "EUPL-1.2",
   "dependencies": {
     "@formio/protected-eval": "^1.2.1",
+    "@formio/react": "^5.2.4-rc.1",
```

Comment:
The SDK currently uses `react-formio` ^4.3 and `formio` 4.12 - I'd stick to those to get an accurate representation. As far as I know, `@react/formio` is the future, but pulls in a newer version of Formio that has known issues with the SDK.